Smartling.API/PublishedFiles/PublishedFilesApiClient.cs

```diff
@@ -40,5 +44,55 @@ public virtual List<PublishedItem> GetRecentlyPublished()
       var result = JsonConvert.DeserializeObject<RecentlyPublished>(response["response"]["data"].ToString());
       return result.items ?? new List<PublishedItem>();
     }
+
+    public virtual RecentlyPublished GetRecentlyPublished(RecentlyPublishedSearch recentlyPublishedSearch)
+    {
+      if (!recentlyPublishedSearch.IsValid())
+      {
+        throw new Exception("Invalid parameters: " + recentlyPublishedSearch.LastError);
+      }
+
+      var uriBuilder = this.GetRequestStringBuilder(string.Format(RecentlyPublishedUrl, projectId, recentlyPublishedSearch.PublishedAfter.ToString("s", System.Globalization.CultureInfo.InvariantCulture)));
+      if(recentlyPublishedSearch.FileUris?.Count > 0)
+      {
+        foreach (var fileUri in recentlyPublishedSearch.FileUris)
+        {
+          uriBuilder.AppendFormat("&{0}={1}", FileUrisParameterName, fileUri);
```

Comment:
Can you please add test for filename with special characters like `test:file&txt`?

Reply:
Added.